modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/AbstractExecutionTest.java

```diff
@@ -103,6 +105,19 @@ public class AbstractExecutionTest extends GridCommonAbstractTest {
     /** */
     protected int nodesCnt = 3;
 
+    /** */
+    protected final int inBufSize;
+
+    /** */
+    protected AbstractExecutionTest() {
+        try {
+            inBufSize = U.field(AbstractNode.class, "IN_BUFFER_SIZE");
```

Comment:
`public static` looks like normal practic.

Reply:
In the same package, protected static works too. Fixed.